backend/battles/tests/test_forms.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from model_mommy import mommy
+
+from battles.forms import CreateTeamForm
+
+
+class TestCreateTeamForm(TestCase):
+    def setUp(self):
+        self.trainer = mommy.make("users.User")
+        self.pokemon_1 = mommy.make("pokemon.Pokemon")
+        self.pokemon_2 = mommy.make("pokemon.Pokemon")
+        self.pokemon_3 = mommy.make("pokemon.Pokemon")
+        self.battle = mommy.make("battles.Battle")
+
+    def test_create_a_team(self):
+        params = {
+            "initial": {"trainer": self.trainer},
+            "data": {
+                "pokemon_1": self.pokemon_1,
+                "pokemon_2": self.pokemon_2,
+                "pokemon_3": self.pokemon_3,
+            },
+        }
+        form = CreateTeamForm(**params)
+        self.assertTrue(form)
+
+    def test_pokemon_exceeds_points_limit(self):
```

Comment:
This test if failing on circleci. Interestingly you are doing the same thing in `test_create_a_team`. I'm confident that you will be able to solve it by debugging the test and looking at `form.errors` before the assert. Use `ipdb` for it, let me know if you need any help!